docs/reference/migration/migrate_6_0/search.asciidoc

```diff
@@ -53,6 +53,10 @@
 
 * The `template` query has been removed. This query was deprecated since 5.0
 
+* The `percolate` query's `document_type` has been deprecated. From 6.0 and later
+  it is no longer required to specify `document_type` parameter on indices with
```

Comment:
let's end the sentence after `parameter` since we plan on not allowing to set this setting at all in 6.x